llvm/test/CodeGen/RISCV/rvv/ctlz-sdnode.ll

```diff
@@ -1231,16 +1231,17 @@ define <vscale x 1 x i64> @ctlz_nxv1i64(<vscale x 1 x i64> %va) {
 ;
 ; CHECK-F-LABEL: ctlz_nxv1i64:
 ; CHECK-F:       # %bb.0:
-; CHECK-F-NEXT:    vsetvli a0, zero, e32, mf2, ta, ma
+; CHECK-F-NEXT:    li a0, 190
```

Comment:
What happens to this part? I think this is a regession.

Reply:
Here `vrsub.vx` and `vzext.vf2` are folded into  `vwsubu.wv`.

Reply:
Thanks for explanation! But I meant that we need a `vmv.v.x` to construct the immediate. The (macro) instructions may not be reduced and we have more vtype toggles now for this `ctlz_nxv1i64` case?

Reply:
Thanks for your advice! It might be possible to use `vwsubu.wx` here. 
I will send a patch on it soon :-)